engine/src/test/java/io/camunda/zeebe/engine/processing/bpmn/compensation/CompensationEventExecutionTest.java

```diff
@@ -797,11 +800,11 @@ public void shouldNotTriggerCompensationIfSubprocessIsNotCompleted() {
             tuple(
                 ValueType.COMPENSATION_SUBSCRIPTION,
                 CompensationSubscriptionIntent.DELETED,
-                "ActivityToCompensate"),
+                "embedded-subprocess"),
             tuple(
                 ValueType.COMPENSATION_SUBSCRIPTION,
                 CompensationSubscriptionIntent.DELETED,
-                "embedded-subprocess"));
+                "ActivityToCompensate"));
   }
 
   @Test
```

Comment:
:wrench: The test case `shouldNotTriggerCompensationOnParentScope` waits 5 seconds at the verification of the compensation subscriptions (line 839). There are only 5 subscription records on the stream, not 6 of them.